#779: Unit tests for Loopback and Firewall classes from core.net

djurod opened the pull request:
- Prepared unit tests for classes LoopbackInterfaceConfigImpl and
  FirewallConfiguration from package org.eclipse.kura.core.net
- Fixed a few smaller issues in class FirewallConfiguration

Signed-off-by: Djuro Drljaca <djuro.drljaca@comtrade.com>

cdealti commented:
You should update the org.eclipse.kura.core.net version in kura.build.properties

Commit:
#779: Unit tests for Loopback and Firewall classes from core.net

- Prepared unit tests for classes LoopbackInterfaceConfigImpl and
  FirewallConfiguration from package org.eclipse.kura.core.net
- Fixed a few smaller issues in class FirewallConfiguration
- Increased org.eclipse.kura.core.net version in kura.build.properties.

Signed-off-by: Djuro Drljaca <djuro.drljaca@comtrade.com>

djurod commented:
Fixed.
